README.md

```diff
@@ -3,42 +3,72 @@
 # React on Rails
 React on Rails integrates Facebook's [React](https://github.com/facebook/react) front-end framework with Rails. Currently, both React v0.14 and v0.13 are supported. See the Rails on Maui [blog post](http://www.railsonmaui.com/blog/2014/10/03/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/) that started it all!
 
-Like the [react-rails](https://github.com/reactjs/react-rails) gem, React on Rails is capable of server-side rendering with fragment caching and is compatible with [turbolinks](https://github.com/rails/turbolinks). Unlike react-rails, which depends heavily on sprockets and jquery-ujs, React on Rails uses [webpack](http://webpack.github.io/) and does not depend on jQuery. While the initial setup is slightly more involved, it allows for advanced functionality such as:
-
-+ [Redux](https://github.com/rackt/redux)
-+ [Webpack dev server](https://webpack.github.io/docs/webpack-dev-server.html) with [hot module replacement](https://webpack.github.io/docs/hot-module-replacement-with-webpack.html)
-+ [Webpack optimization functionality](https://github.com/webpack/docs/wiki/optimization)
-+ **(Coming Soon)** [React Router](https://github.com/rackt/react-router)
-
-See the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/) for an example of a live implementation and code.
-
-### Including your React Component in your Rails Views
+## Including your React Component in your Rails Views
 Please see [Getting Started](#getting-started) for how to set up your Rails project for React on Rails if you have not already done so.
 
-+ *Normal Mode (JavaScript is rendered on client):*
++ *Normal Mode (React component will be rendered on client):*
 
   ```erb
   <%= react_component("HelloWorldApp", @some_props) %>
   ```
-+ *Server-Side Rendering:*
++ *Server-Side Rendering (React component is first rendered into HTML on the server):*
 
   ```erb
   <%= react_component("HelloWorldApp", @some_props, prerender: true) %>
   ```
 
-+ The `component_name` parameter here would be a string matching the name you used when globally exposing your React component.
-+ `@some_props` can be either a hash or JSON string or {} if you have no props. This will make the data available in your component:
++ The `component_name` parameter is a string matching the name you used when globally exposing your React component. So, in the above examples, if you had a React component named "HelloWorld," you would set `window.HelloWorldApp = HelloWorld` in your JavaScript. Exposing your component in this way is how React on Rails is able to reference your component from a Rails view. You can expose as many components as you like, as long as their names do not collide.
```

Comment:
See below for the details of how you expose your components via the react_on_rails webpack configuration.